packages/utils/src/lib/report-to-md.ts

```diff
@@ -105,73 +140,37 @@ function reportToCategoriesSection(report: ScoredReport): string {
       categoryDocs +
       categoryScore +
       NEW_LINE +
+      groups +
       NEW_LINE +
-      refs
+      audits
     );
   }, '');
 
   return h2('🏷 Categories') + NEW_LINE + categoryDetails;
 }
 
-function auditRefItemToCategorySection(
-  refSlug: string,
-  refPlugin: string,
-  plugins: ScoredReport['plugins'],
-): string {
-  const plugin = plugins.find(({ slug }) => slug === refPlugin) as PluginReport;
-  const pluginAudit = plugin?.audits.find(({ slug }) => slug === refSlug);
-
-  if (!pluginAudit) {
-    throwIsNotPresentError(`Audit ${refSlug}`, plugin?.slug);
-  }
-
+function auditItemToCategorySection(audit: WeighedAuditReport): string {
   const auditTitle = link(
-    `#${slugify(pluginAudit.title)}-${slugify(plugin.title)}`,
-    pluginAudit?.title,
+    `#${slugify(audit.title)}-${slugify(audit.plugin)}`,
+    audit?.title,
   );
-
-  return (
-    li(
-      `${getSquaredScoreMarker(pluginAudit.score)} ${auditTitle} (_${
-        plugin.title
-      }_) - ${getAuditResult(pluginAudit)}`,
-    ) + NEW_LINE
+  return li(
+    `${getSquaredScoreMarker(audit.score)} ${auditTitle} (_${
+      audit.plugin
+    }_) - ${getAuditResult(audit)}`,
   );
 }
 
-function groupRefItemToCategorySection(
-  refSlug: string,
-  refPlugin: string,
-  plugins: ScoredReport['plugins'],
+function groupItemToCategorySection(
+  group: EnrichedScoredAuditGroupWithAudits,
 ): string {
-  const plugin = plugins.find(({ slug }) => slug === refPlugin) as PluginReport;
-  const group = plugin?.groups?.find(
-    ({ slug }) => slug === refSlug,
-  ) as EnrichedScoredAuditGroup;
   const groupScore = Number(formatReportScore(group?.score || 0));
-
-  if (!group) {
-    throwIsNotPresentError(`Group ${refSlug}`, plugin?.slug);
-  }
-
   const groupTitle = li(
-    `${getRoundScoreMarker(groupScore)} ${group.title} (_${plugin.title}_)`,
+    `${getRoundScoreMarker(groupScore)} ${group.title} (_${group.plugin}_)`,
```

Comment:
Same here, make sure this is the plugin's title, please :pray:

Reply:
Fixed